andela/ah-orcas-frontend
# #161254626 Add/view tags

#### What does this PR do?
Allow user to view and create articles
#### Description of Task to be completed?
```Add tags to article```
``` view tags```
#### How should this be manually tested?
- Clone this repository
`git clone https://github.com/andela/ah-orcas-frontend.git`
- Change the working directory to the project folder
`cd ah-orcas-frontend`
- Checkout to the appropriate branch
`git checkout ft-view-tags-161254626`
- Install the required dependencies:
     `npm start`
should be able to add tags upon article creation
 
#### What are the relevant pivotal tracker stories?
[#161254626](https://www.pivotaltracker.com/story/show/161254626)

![image](https://user-images.githubusercontent.com/30722290/50295946-5c2d9580-048a-11e9-90c9-3fdfe6c20ada.png)
![image](https://user-images.githubusercontent.com/30722290/50295959-62237680-048a-11e9-9113-12f7b9021581.png)


#### Checklist:
